src/main/java/dev/tycho/stonks/command/base/LambdaSubCommand.java

```diff
@@ -0,0 +1,30 @@
+package dev.tycho.stonks.command.base;
+
+import org.bukkit.entity.Player;
+
+import java.util.List;
+
+public class LambdaSubCommand extends SubCommand {
```

Comment:
There is no need for this class, just use SimpleSubCommand as it will automatically use a lambda